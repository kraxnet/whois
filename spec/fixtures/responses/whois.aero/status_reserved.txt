Name is restricted from registration
