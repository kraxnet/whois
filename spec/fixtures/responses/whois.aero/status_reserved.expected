#disclaimer
  %s == nil


#domain
  %s == nil

#domain_id
  %s == nil


#status
  %s == :reserved

#available?
  %s == false

#registered?
  %s == true

#reserved?
  %s == true


#created_on
  %s == nil

#updated_on
  %s == nil

#expires_on
  %s == nil


#registrar
  %s == nil

#registrant_contacts
  %s %CLASS{array}
  %s == []

#admin_contacts
  %s %CLASS{array}
  %s == []

#technical_contacts
  %s %CLASS{array}
  %s == []


#nameservers
  %s %CLASS{array}
  %s == []
